cypress/e2e/cloud/deepLinks.test.ts

```diff
@@ -132,12 +132,6 @@ describe('Deep linking', () => {
       cy.visit('/me/tasks')
       cy.location('pathname').should('eq', `/orgs/${org.id}/tasks`)
 
-      cy.visit('/me/telegraf-mqtt')
-      cy.location('pathname').should(
-        'eq',
-        `/orgs/${org.id}/load-data/telegraf-plugins/mqtt_consumer`
```

Comment:
I think we still want the telegraf plugin included for the reason I mention in a later comment, since it's distinct from a direct MQTT subscription in Cloud2.